### PR TITLE
feat: wishlist

### PR DESCRIPTION
**Pull request by aniketsingh13**
Screenshot 2022-02-16 181908

### CODE REVIEW

File: wishlist/wishlist.html [hunk 1]
```diff
@@ -0,0 +1,122 @@
...
+    </nav>
+    <!--main body-->
+    <h1 class="whislist-heading mt-l f-xl font-xl">My Wishlist</h1>
+    <div class="whislist-product flex flex-center mb-l  mt-l">
+        <div class="cards_withBadge">
+            <div class="header">
+                <image class="card-image_one" src="../assets/shopping.webp" alt="controler-image">
+                    <span class="card-icon p-xs br-s"><i class="far fa-heart"></i></span>
+            </div>
+            <div class="body  flex flex-center flex-column">
```
Comment (SanketDhabarde) [inline wishlist/wishlist.html:46]: give some margin between footer and body

**Review by SanketDhabarde — CHANGES_REQUESTED** [review]
Just add some spacing between the body and footer of the card it will look good. and use prettier for proper indentation.

File: wishlist/wishlist.html (hunk 1 above)
Comment (aniketsingh13) [reply wishlist/wishlist.html:46]: will do that

**Review by SanketDhabarde — APPROVED** [review]
looks good to me

File: wishlist/wishlist.html [hunk 2]
```diff
@@ -0,0 +1,122 @@
...
+<body>
+    <nav class="nav-conatainer flex  flex-row p-xs">
+        <div class="nav-brand font-xl f-l"><a class="nav-brand-a" href="">PlayX</a></div>
+        <form class="p-s">
+            <input class="nav-input font-xl f-m" type="search" name="search" placeholder="Search..">
+        </form>
+        <ul class="nav-right flex p-s">
+            <li class="pl-m"><button class="nav-login-btn f-s"><a class="" href="">Login</a></button></li>
+            <li class="nav-love-icon pl-l">
+                <a class="nav-loveicon-a" href=""><i class="far fa-heart f-l"></i></a>
```
Comment (shubhamkumar648) [inline wishlist/wishlist.html:27]: never keep href attributes empty use #

File: wishlist/wishlist.html [hunk 3]
```diff
@@ -0,0 +1,122 @@
...
+        <div class="cards_withBadge">
+            <div class="header">
+                <image class="card-image_one" src="../assets/shopping.webp" alt="controler-image">
+                    <span class="card-icon p-xs br-s"><i class="far fa-heart"></i></span>
+            </div>
+            <div class="body  flex flex-center flex-column">
+                <div class="p-xss"> wireless controller</div>
+                <div class="f-m font-xs">रु5690</div>
+            </div>
+            <div class="footer p-xs">
```
Comment (shubhamkumar648) [inline wishlist/wishlist.html]: you can also use span, small element tag instead of div

File: wishlist/wishlist.html [hunk 4]
```diff
@@ -0,0 +1,122 @@
...
+                <p>Add to cart</p>
+            </div>
+        </div>
+        <div class="cards_withBadge">
+            <div class="header">
+                <image class="card-image_one" src="../assets/shopping.webp" alt="controler-image">
+                    <span class="card-icon p-xs br-s"><i class="far fa-heart"></i></span>
+            </div>
+            <div class="body  flex flex-center flex-column">
```
Comment (shubhamkumar648) [inline wishlist/wishlist.html:98]: good use of class name !

**Review by shubhamkumar648 — APPROVED** [review]
code looks clean and good UI design , you can merge

File: wishlist/wishlist.html (hunk 2 above)
Comment (aniketsingh13) [reply wishlist/wishlist.html:27]: goona use that

File: wishlist/wishlist.html (hunk 3 above)
Comment (aniketsingh13) [reply wishlist/wishlist.html]: there a reason for using it

File: wishlist/wishlist.html (hunk 4 above)
Comment (aniketsingh13) [reply wishlist/wishlist.html:98]: thanks for the review

**Pull request merged**